src/api_parser.ts

```diff
@@ -0,0 +1,40 @@
+import { AB_CONTRACTS } from './utils';
+const { createClient, gql } = require('@urql/core');
+const parse = require('node-html-parser').parse;
+const fetch = require('node-fetch');
+const API_URL = 'https://api.thegraph.com/subgraphs/name/artblocks/art-blocks';
+
+const client = createClient({
+  url: API_URL,
+  fetch: fetch,
+  fetchOptions: () => ({
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  }),
+});
+
+const _getPBABContracts = gql`
+  query getPBABContracts($ids: [ID]!) {
+    contracts(where: { id_not_in: $ids }) {
```

Comment:
naming nit: Can we either rename this to be something generic like `_getContracts` with arg `$ids_not_in`, or maybe keep the function name, remove the args, and just automatically include `${AB_CONTRACTS}` in the gql string?

Reply:
done!